dev/integration_tests/android_host_app/app/src/main/AndroidManifest.xml

```diff
@@ -4,7 +4,8 @@
     package="io.flutter.add2app">
 
     <application android:allowBackup="false"
-        tools:ignore="GoogleAppIndexingWarning,MissingApplicationIcon">
+        tools:ignore="GoogleAppIndexingWarning,MissingApplicationIcon"
+        tools:replace="android:appComponentFactory">
```

Comment:
What does this do?

Reply:
committed this by mistake. Thanks for catching it up.